pkg/spec/spec.go

```diff
@@ -188,6 +188,47 @@ func CreateConfigToOCISpec(config *CreateConfig) (*spec.Spec, error) { //nolint
 		}
 	}
 
+	if strings.HasSuffix(config.Command[0], "init") ||
+		strings.HasSuffix(config.Command[0], "systemd") {
+		options := []string{"rw", "private", "noexec", "nosuid", "nodev"}
+		for _, i := range []string{"/run", "/run/lock", "/sys/fs/cgroup/systemd"} {
+			tmpfsMnt := spec.Mount{
+				Destination: i,
+				Type:        "tmpfs",
+				Source:      "tmpfs",
+				Options:     append(append(options, "tmpcopyup"), "size=65536k"),
+			}
+			g.AddMount(tmpfsMnt)
+		}
+		for _, i := range []string{"/tmp", "/var/lib/journal"} {
+			tmpfsMnt := spec.Mount{
+				Destination: i,
+				Type:        "tmpfs",
+				Source:      "tmpfs",
+				Options:     append(options, "tmpcopyup"),
+			}
+			g.AddMount(tmpfsMnt)
+		}
+	}
+	for _, i := range config.Tmpfs {
+		// Default options if nothing passed
+		options := []string{"rw", "private", "noexec", "nosuid", "nodev", "size=65536k"}
+		spliti := strings.SplitN(i, ":", 2)
+		if len(spliti) > 1 {
+			if _, _, err := mount.ParseTmpfsOptions(spliti[1]); err != nil {
+				return nil, err
+			}
+			options = strings.Split(spliti[1], ",")
+		}
+		tmpfsMnt := spec.Mount{
+			Destination: spliti[0],
+			Type:        "tmpfs",
+			Source:      "tmpfs",
+			Options:     append(options, "tmpcopyup"),
+		}
+		g.AddMount(tmpfsMnt)
+	}
+
 	for _, i := range config.Tmpfs {
```

Comment:
Is this still supposed to be here? Looks duplicated

Reply:
Yup, removed.